releasenotes/v1.11.0-alpha.0.md

```diff
@@ -0,0 +1,107 @@
+- Change deprecated ginkgo functionality (#2640)
+- Bump sigs.k8s.io/kustomize/api from 0.19.0 to 0.20.0 in /test (#2643)
+- Bump the github-actions group with 3 updates (#2639)
+- Limit API listing to 200 at a time via pagination (#2646)
+- Bump sigs.k8s.io/kustomize/kyaml from 0.19.0 to 0.20.0 in /test (#2642)
+- Bump sigs.k8s.io/kustomize/kustomize/v5 from 5.6.0 to 5.7.0 in /hack/tools (#2641)
+- Bump github.com/docker/docker from 28.3.0+incompatible to 28.3.2+incompatible in /test (#2650)
+- Bump github.com/go-viper/mapstructure/v2 from 2.2.1 to 2.3.0 in /test (#2636)
+- Bump github.com/docker/docker from 28.2.2+incompatible to 28.3.0+incompatible in /test (#2635)
+- Adopt md_rollout test from CAPI (#2584)
+- Bump the kubernetes group to v0.32.6 (#2632)
+- Bump IRSO to v0.4.0 in /test (#2634)
+- Adopt md_scale test from CAPI (#2606)
+- Bump sigs.k8s.io/yaml from 1.4.0 to 1.5.0 (#2633)
+- Fix deleting BMHs in md_remediation (#2618)
+- Bump github.com/metal3-io/cluster-api-provider-metal3/api from 1.10.0 to 1.10.1 in /hack/fake-apiserver (#2626)
+- Bump CAPI to v1.10.3 (#2624)
+- Uplift Kubernetes to 1.33 in envtest (#2535)
+- E2E: longer timeout for cert-manager (#2615)
+- Bump github.com/metal3-io/cluster-api-provider-metal3 from 1.10.0 to 1.10.1 in /hack/fake-apiserver (#2625)
+- Fixing remediation tests (#2616)
+- nilnil linter (#2579)
+- bump golang to 1.24.4 (#2609)
+- fix ephemeral worker/cp count (#2607)
+- Bump golang.org/x/crypto from 0.38.0 to 0.39.0 in /test (#2604)
+- Fix image source when running E2E tests from ironic-image or BMO (#2559)
+- Adopt k8s conformance test from CAPI (#2568)
+- Bump github.com/cloudflare/circl from 1.3.7 to 1.6.1 in /test (#2598)
+- Add vars for feature tests and improve the script (#2596)
+- remove obsolete markdownlinkcheck config (#2597)
+- Use reusable link-checks and add .lycheeignore (#2594)
+- Bump github.com/metal3-io/baremetal-operator/apis from 0.10.1 to 0.10.2 in /test (#2590)
+- Bump github.com/metal3-io/baremetal-operator/apis from 0.10.1 to 0.10.2 in /hack/fake-apiserver (#2589)
+- Bump github.com/docker/docker from 28.2.0+incompatible to 28.2.2+incompatible in /test (#2591)
+- Bump github.com/metal3-io/baremetal-operator/apis from 0.10.1 to 0.10.2 (#2588)
+- Apply BMH from CAPM3 e2e feature tests (#2583)
+- Bump the kubernetes group v0.32.5 (#2561)
+- Bump the github-actions group with 2 updates (#2581)
+- Bump github.com/go-logr/logr from 1.4.2 to 1.4.3 (#2575)
+- Bump CAPI to v1.10.2 (#2562)
+- Bump github.com/docker/docker from 28.1.1+incompatible to 28.2.0+incompatible in /test (#2576)
+- Updating main branch clusterctl tests (#2560)
+- Bump golang to v1.24 (#2569)
+- Apply BMH from CAPM3 during test and skip BMH creation from dev-env (#2505)
+- Fix ipam deploy in clusterctl tests (#2554)
+- Enable exhaustive linter (#2544)
+- Fix typo in bmo deployment overlay kustomization file name (#2556)
+- Bump CAPI to v1.10.1 (#2537)
```

Comment:
Should we use superseded in alpha or not, I used it but I don't remember actually if we have a policy for that or not.

Reply:
Okay we have discussed offline that according to existing code the block is not generated for Beta and RC so it should not be generated for Alpha either but the exclusion of the alpha release is/was missing from the repos.

We can revisit this topic later.